packages/navbar/src/utils/get-border-radius-styling.ts

```diff
@@ -8,25 +8,25 @@ export const getBorderRadiusStyling = (orientation?: Orientation, isFirst?: bool
 
   if (orientation === 'inline') {
     if (isFirst) {
-      topLeft = '5px';
-      bottomLeft = '5px';
+      topLeft = '3px';
```

Comment:
This is to match the rounded corners of a card... Should we have an option to modify these attributes through the theme?